sessions/js-variables-and-numbers/social-media-post/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+		<title>Social Media Post</title>
+		<link rel="stylesheet" href="./css/styles.css" />
+		<script src="./js/index.js" defer></script>
+	</head>
+	<body>
+		<header>
+			<h1>Welcome to this Exercise!</h1>
```

Comment:
Exercise would be in lowercase here. The capitalizing of nouns in English is a little bit ... weird. It wouldn't apply here, though.

Reply:
Yes! This is somethig we should fix in a couple of exercises using this note